packages/react-components/review/beta/react-components.api.md

```diff
@@ -415,6 +416,7 @@ export const DEFAULT_COMPONENT_ICONS: {
     SendBoxSendHovered: JSX.Element;
     VideoTileMicOff: JSX.Element;
     BackSpace: JSX.Element;
+    iconBackdrop: JSX.Element;
```

Comment:
Don't use an icon for the backdrop, just create a circle from css

Reply:
Semantic question, if I'm contoso overriding an icon - should the circle remain? or should our "icon" be circle+camera?

Reply:
I actually like this being circle + camera, since I am already applying the scaling in our own component changing this to just be the two icons together I think would be the way to go. will make the update!

Reply:
Will do this in PR introducing the sparkle icon: https://github.com/Azure/communication-ui-library/pull/2307/files